### PR TITLE
Add v0.1.3 NEWS

### PR DESCRIPTION
**Pull request by vcaputo**
Adds necessary ssh keys support to the ec2 provider.

### CODE REVIEW

**Comment by crawford** [comment]
We can't merge version bumps; they must be pushed directly to master. Otherwise, you'll have multiple repo states (d36174f, 4cd36a3, and the merge) which have the same version number. Additionally, it's not possible to enforce that no other changes are merged before the version is again bumped to "0.1.3+git".

I also tried the latest code with EC2 keys and it still doesn't work.

**Comment by vcaputo** [comment]
@crawford ok this is pared back to just the NEWS update, and we'll tag the merge commit as v0.1.3

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,16 @@
+11-Aug-2015 IGNITION v0.1.3
+
+  Features
+
+    - Add support for ssh keys on EC2
+    - Derive version from git describe at build time
+    - Use bash build and test scripts instead of make
```
Comment (crawford) [inline NEWS]: I'm not sure if this is a feature

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+
+    - Add support for ssh keys on EC2
+    - Log version at runtime
+
+  Bug Fixes
+
+    - Log ssh keys as they are added
+    - Various small cleanups
+
+  Build system changes
```
Comment (crawford) [inline NEWS]: Title case

**Comment by crawford** [comment]
Please make sure the merge has the same comment as this commit (I think you can just edit this title and it will use that).

**Comment by crawford** [comment]
LGTM

**Pull request merged**